install/migrations/update_10.0.x_to_11.0.0/form.php

```diff
@@ -329,6 +331,9 @@
     $migration->addField("glpi_forms_comments", "forms_sections_uuid", "string");
     $migration->addKey("glpi_forms_comments", "uuid", type: 'UNIQUE');
     $migration->addKey("glpi_forms_comments", "forms_sections_uuid");
+
+    $migration->addField("glpi_helpdesks_tiles_profiles_tiles", "rank", "int");
+    $migration->addKey("glpi_helpdesks_tiles_profiles_tiles", "rank");
```

Comment:
unicity key seems missing here

Reply:
This is temporary code so I got lazy and avoided adding the unique key here as it would mean setting correct rank for each existing rows and I wanted to avoid wasting time on that since the code would be deleted anyway.

Since its getting removed in #18694, I'll drop the change on these lines to avoid conflicts.